### ISSUE TITLE
Typography - Colour Contrast

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#7]
**Requirement**: There must be an adequate colour contrast ratio for text.
- Text that is less than 18 point (or 14 point if bold) has a contrast ratio of at least 4.5:1.
- Text that is greater than 18 point has a contrast ration of at least 3:1.

**Exception**: Text that is part of a logo or brand name has no minimum contrast requirement.

Note - ‘Large scale’ is defined as 24px (18 point) text or 19px (14 point) bold text. It is acknowledged that point sizes are different for different fonts, but WCAG states ‘“18 point” and “bold” can both have different meanings in different fonts but, except for very thin or unusual fonts, they should be sufficient. Since there are so many different fonts, the general measures are used and a note regarding fancy or thin fonts is included.’ (From Understanding SC 1.4.3. http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/visual-audio-contrast-contrast.html)
